src/renderer/components/editorWithTabs/sourceCode.vue

```diff
@@ -137,7 +138,8 @@
   .source-code .CodeMirror-activeline-gutter {
     background: #F2F6FC;
   }
-  .dark {
+  .dark,
```

Comment:
Sorry, you need write this way `.dark.source-code`.

Because we didn't add `scoped` to `<style>` tag, [component-scoped-css](https://vuejs.org/v2/guide/comparison.html#Component-Scoped-CSS) So this rule is globally, add `.source-code` classname just make  this rule affect this component.

Reply:
OK, than this has to be changed in other files too.